operations/md_ops/transcript.py

```
@@ -0,0 +1,311 @@
+                            time_str = time_m.group(1)
+                        else:
+                            content_parts.append(text)
+                if user_id and (time_str or content_parts):
+                    transcript.append(
+                        {
+                            "user_id": user_id,
+                            "time": time_str,
+                            "content": "".join(content_parts).strip(),
+                        }
+                    )
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)